### ISSUE TITLE
MavenDiagnosticParticipant returns null range

### ISSUE THREAD

**Issue by rgrunber** [eclipse/lemminx-maven#458]
```
Jul. 26, 2023 8:29:56 P.M. org.eclipse.lemminx.services.XMLDiagnostics doExtensionsDiagnostics
SEVERE: Error while processing diagnostics for the participant 'org.eclipse.lemminx.extensions.maven.participants.diagnostics.MavenDiagnosticParticipant'.
java.lang.NullPointerException: Property must not be null: range
	at org.eclipse.lsp4j.util.Preconditions.checkNotNull(Preconditions.java:29)
	at org.eclipse.lsp4j.Diagnostic.<init>(Diagnostic.java:91)
	at org.eclipse.lsp4j.Diagnostic.<init>(Diagnostic.java:96)
	at org.eclipse.lsp4j.Diagnostic.<init>(Diagnostic.java:102)
	at org.eclipse.lemminx.extensions.maven.participants.diagnostics.DiagnosticRequest.createDiagnostic(DiagnosticRequest.java:40)
	at org.eclipse.lemminx.extensions.maven.participants.diagnostics.PluginValidator.validateConfigurationElement(PluginValidator.java:154)
	at org.eclipse.lemminx.extensions.maven.participants.diagnostics.PluginValidator.validateConfiguration(PluginValidator.java:95)
	at org.eclipse.lemminx.extensions.maven.participants.diagnostics.MavenDiagnosticParticipant.lambda$doDiagnostics$3(MavenDiagnosticParticipant.java:84)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179)
	at java.base/java.util.HashMap$EntrySpliterator.forEachRemaining(HashMap.java:1850)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
	at org.eclipse.lemminx.extensions.maven.participants.diagnostics.MavenDiagnosticParticipant.doDiagnostics(MavenDiagnosticParticipant.java:85)
	at org.eclipse.lemminx.services.XMLDiagnostics.doExtensionsDiagnostics(XMLDiagnostics.java:67)
	at org.eclipse.lemminx.services.XMLDiagnostics.doDiagnostics(XMLDiagnostics.java:49)
	at org.eclipse.lemminx.services.XMLLanguageService.doDiagnostics(XMLLanguageService.java:190)
	at org.eclipse.lemminx.services.XMLLanguageService.publishDiagnostics(XMLLanguageService.java:204)
	at org.eclipse.lemminx.XMLTextDocumentService.validate(XMLTextDocumentService.java:716)
	at org.eclipse.lemminx.XMLTextDocumentService.lambda$new$2(XMLTextDocumentService.java:207)
	at org.eclipse.lemminx.commons.ModelValidatorDelayer.lambda$validateWithDelay$0(ModelValidatorDelayer.java:69)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
```

**Comment by vrubezhny** [comment]
Thanks for reporting!

This is looks like a result of preceding BLE-exception like the one from #460

**Comment by vrubezhny** [comment]
I believe this should be fixed now with #462. Please, feel free to reopen if you believe it's not.
